src/Form/Type/ChoiceFieldMaskType.php

```diff
@@ -26,14 +26,17 @@ class ChoiceFieldMaskType extends AbstractType
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
         $sanitizedMap = [];
+        $allFieldNames = [];
         foreach ($options['map'] as $value => $fieldNames) {
-            foreach ($fieldNames as $fieldName) {
-                $sanitizedMap[$value][] =
-                    str_replace(['__', '.'], ['____', '__'], $fieldName);
+            if (is_array($fieldNames)) {
```

Comment:
I am not sure about this if, the ideal would be to use is_iterable but that function is only php 7.1. The foreach also works with objects.

Reply:
https://github.com/symfony/polyfill-php71/blob/master/Php71.php#L23

What about adding this second condition?